src/client/settings/templates/settings-cache.html

```diff
@@ -14,7 +14,7 @@
 
         <md-input-container>
             <md-checkbox ng-model="settings.config.deleteTmpOnClose">
-                Clear Tmp Folder after closing app?
```

Comment:
@xaiki btw, my email is zridouh@live.com for the slack :)

I agree with @hadim about the hackability of the project, and I don't think that the app should break should break from the lack of translation. Also, wouldn't it be easier for people to contribute and for simplicities sake to have it inside a separate repo that we can just pull into all the butter projects.